packages/wellbeing-mock-data/wellbeing-mock-data.js

```diff
@@ -130,20 +130,106 @@ function createMockActivities() {
   }
 }
 
+
+function getRandomHomeButExcludeCurrent(current_id) {
```

Comment:
What current ID is this? Lets make it explicit, e.g.

- currentHomeId
- currentResidentId

Note: JavaScript variables in this project use camelCase, so lets stick to that convention.